### ISSUE TITLE
Multiple faces training

### ISSUE THREAD

**Issue by venditto** [iperov/DeepFaceLab#713]
Hello, it's possible training 2 faces in one time?

**Comment by capsmanofficial** [comment]
you ask a little too much I think. : D

**Comment by jacobfnl** [comment]
Absolutely, if you have a second GPU, you can run two instances. However, you would definitely need to run using custom parameters on the command line, because the source identities would need to be in separate (custom named) directories and those paths passed specifically for the faces you wish to train.

**Comment by test1230-lab** [comment]
close issue

**Comment by ghost** [comment]
This is not an issue (or the issue is not related to deepfacelab), please close it.
